README.rst

```diff
@@ -44,27 +44,28 @@ Please read https://docs.conan.io/en/latest/installation.html
 From binaries
 -------------
 
-We have installers for `most platforms here <http://conan.io>`__ but you
+We have installers for `most platforms here <http://conan.io>`_ but you
 can run **conan** from sources if you want.
 
 From pip
```

Comment:
Actually, I think that we should completely remove the "user" installation part of the readme, and redirect to the Conan docs, because there are lots of tips about it there: virtualenvs, possible errors, recommendations...

I would leave here only the "developer" documentation. How to install for working on the code, and run tests.

WDYT?

Reply:
Changed. This is how it looks like now. I never know how to use the `TL;DR` but I think we should keep here the `pip install conan` line.

![image](https://user-images.githubusercontent.com/1406456/78346879-f80be380-759f-11ea-8345-d173e8cc78d7.png)